plugins/webhooks/github.js

```diff
@@ -171,7 +171,7 @@ function pullRequestEvent(request, reply) {
     const payload = request.payload;
     const action = hoek.reach(payload, 'action');
     const prNumber = hoek.reach(payload, 'pull_request.number');
-    const repository = hoek.reach(payload, 'pull_request.base.repo.ssh_url');
+    const repository = hoek.reach(payload, 'pull_request.base.repo.ssh_url').toLowerCase();
```

Comment:
Part of me thinks you should be running scmUrl, and prRef through the `scmPlugin.formatScmUrl` https://github.com/screwdriver-cd/scm-github/blob/master/index.js#L71-L79